src/qibolab/pulses/pulse.py

```diff
@@ -24,10 +22,8 @@ class PulseType(Enum):
 
 @dataclass
 class Pulse:
-    """A class to represent a pulse to be sent to the QPU."""
+    """Representation of a pulse to be sent to the QPU."""
```

Comment:
```suggestion
    """A pulse to be sent to the QPU."""
```

Since all objects are representations of something else, maybe that's just enough